saleor/graphql/order/mutations/draft_orders.py

```diff
@@ -255,6 +257,18 @@ def update_user_fields(cls, order):
             except User.DoesNotExist:
                 order.user = None
 
+    @staticmethod
+    def avalara_active():
+        manager = get_plugins_manager()
```

Comment:
The plugin manager is available in the request context: `info.context.plugins`, we should use it to save a database query which `get_plugins_manager` makes.